web/private-chat-subdomain/src/main/kotlin/com/example/ajax/infrastructure/entity/RedisPrivateChatRoom.kt

```diff
@@ -1,4 +1,4 @@
-package com.example.ajax.infrastructure.redis.entity
+package com.example.ajax.infrastructure.entity
```

Comment:
the previous version was better. In package `*.infrastructure.entity`

Reply:
Agree